### ISSUE TITLE
consistent naming: encryption_algorithm should be encryptionAlgorithm

### ISSUE THREAD

**Issue by recipe** [onelogin/php-saml#514]
The `encryption_algorithm` option should be named as `encryptionAlgorithm` to be consistent.
https://github.com/onelogin/php-saml/commit/7b4d4a6df94a782b4688ebb3d9c6a1334258376c

Internally we keep `onelogin/php-saml` configuration with `snake_case` keys to be consistent with our own configuration naming conventions. Before creating the `php-saml` instance we transform all `php-saml` related keys to `CamelCase` and use this array to satisfy `php-saml` naming conventions. Now we will have to create a special rule for this single option to not to convert to `CamelCase`.

**Comment by pitbulk** [comment]
@recipe   You are right, we should fix it by deprecating the use of encryption_algorithm (basically remove its use on demos and documentation) and in order to keep the backward compatibility read the possible values from encryption_algorithm and encryptionAlgorithm giving priority to the latest.

Can you contribute with a PR?
